Kopernicus/Kopernicus/RuntimeUtility/StarLightSwitcher.cs

```diff
@@ -50,8 +50,8 @@ static StarComponent()
         void Start()
         {
             // Find the celestial body we are attached to
-            celestialBody = PSystemManager.Instance.localBodies.First(body => body.scaledBody == gameObject);
-            Logger.Default.Log("StarLightSwitcher.Start() => " + celestialBody.bodyName);
+            celestialBody = PSystemManager.Instance.localBodies.FirstOrDefault(body => body.scaledBody == gameObject);
```

Comment:
While I agree that this shouldn't throw the message it currently throws, it should throw something. Maybe something like this

```csharp
if (celestialBody == null)
    throw new InvalidOperationException("The StarComponent isn't attached to the ScaledSpace object of a body.");
```

Reply:
the problem is that in the RnD all stars have a StarComponent

I don't think they should thrown an error everytime you enter the RnD

Reply:
Ok, then leave it that way